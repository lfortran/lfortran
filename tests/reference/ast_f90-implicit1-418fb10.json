{
    "basename": "ast_f90-implicit1-418fb10",
    "cmd": "lfortran --show-ast-f90 --no-indent --no-color {infile}",
    "infile": "tests/implicit1.f90",
    "infile_hash": "055efd6feee3a318ee48bce964a4223461719dc2c772db80c7440f82",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "ast_f90-implicit1-418fb10.stdout",
    "stdout_hash": "da90bd9449cc668dc87b7e3b46163c6f97c78649065cafd5d5d2bf68",
    "stderr": "ast_f90-implicit1-418fb10.stderr",
    "stderr_hash": "8fb7cba70f8f53adeb61cebe9b2ec9569553e1c2d1593891217dd169",
    "returncode": 0
}