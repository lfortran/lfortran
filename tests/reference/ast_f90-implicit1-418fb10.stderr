style suggestion: Use real(8) instead of real*8
  --> tests/implicit1.f90:13:10
   |
13 | implicit real*8 (a-h,o-z)
   |          ^^^^^^ help: write this as 'real(8)'

style suggestion: Use integer(8) instead of integer*8
  --> tests/implicit1.f90:25:10
   |
25 | implicit integer*8 (i,j-l,m,n)
   |          ^^^^^^^^^ help: write this as 'integer(8)'

style suggestion: Use integer(4) instead of integer*4
  --> tests/implicit1.f90:27:10
   |
27 | IMPLICIT INTEGER*4 (C, D-x)
   |          ^^^^^^^^^ help: write this as 'integer(4)'

style suggestion: Use logical(4) instead of logical*4
  --> tests/implicit1.f90:32:10
   |
32 | implicit logical*4 (l, u-z)
   |          ^^^^^^^^^ help: write this as 'logical(4)'

style suggestion: Use complex(8) instead of complex*8
  --> tests/implicit1.f90:38:10
   |
38 | implicit complex*8 (z)
   |          ^^^^^^^^^ help: write this as 'complex(8)'
